src/postprocess/dos.jl

```diff
@@ -68,65 +68,185 @@ function compute_ldos(scfres::NamedTuple; ε=scfres.εF, kwargs...)
 end
 
 """
-Projected density of states at energy ε for an atom with given i and l.
+    compute_pdos(εs, basis, ψ, eigenvalues; [positions, smearing, temperature])
+
+Compute the projected density of states (PDOS) for all atoms and orbitals.
+
+Overview of inputs: 
+- `εs`: Vector of energies at which the PDOS will be computed
+
+Overview of outputs:
+- `pdos`: 3D array of PDOS, pdos[iε_idx, iproj, σ] = PDOS at energy εs[iε_idx] for projector iproj and spin σ
+- `projector_labels` : Vector of tuples (iatom, species, label, n, l, m) for each projector, that maps the iproj index to the corresponding atomic orbital. 
```

Comment:
I agree with everything, but some lines are still to long (this is just one example). The loose limit is 92 characters. If it helps readability, being a bit longer is fine (i.e. we are not super strict about it). But these two lines are both well over 100 characters ... that's not ok.